src/popup/router/routes.ts

```diff
@@ -205,10 +206,11 @@ export const routes: WalletAppRouteConfig[] = [
       {
         path: 'details/',
         component: AccountDetailsMultisig,
+        redirect: { name: ROUTE_MULTISIG_DETAILS_TRANSACTIONS },
```

Comment:
Please avoid using redirects. The child page, that should be opened as the default should have the path set as empty string.